test/test_hex_two_headed_net.py

```diff
@@ -61,10 +60,12 @@ def run_test():
             }, f)
 
         logging.info("Running self play and generating new models...")
-        subprocess.check_call([
-            "python", PYTHON_MAIN,
-            "--config", CONFIG_FILE],
-            stderr=subprocess.STDOUT)
+        python = sys.executable
+        subprocess.check_call(" ".join(

```

Comment:
I the " ".join is necessary? i think you can just pass array of args

Reply:
* The `join` is necessary for `shell=True` to work, and 
* `shell=True` is necessary in order to find the `cargo` executable in the Github slaves (I don't know why the shell finds it but the Python subprocess mechanism doesn't)

Reply:
K